File: app_i/src/main/java/com/recoveryrecord/surveyandroid/example/ui/setting/SettingsActivity.java
package com.recoveryrecord.surveyandroid.example.ui.setting;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

import com.recoveryrecord.surveyandroid.example.NewsHybridActivity;
import com.recoveryrecord.surveyandroid.example.R;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.settings_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager()
                    .beginTransaction()
                    .replace(R.id.settings, new SettingsFragment())
                    .commit();
        }
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
        }
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                //do whatever
                Intent intent_back = new Intent(SettingsActivity.this, NewsHybridActivity.class);
                startActivity(intent_back);
                return true;
            default:
//                Intent intent_backb = new Intent(SettingsActivity.this, NewsHybridActivity.class);
//                startActivity(intent_backb);
//                return true;
                return super.onOptionsItemSelected(item);
        }
    }
    @Override
    public void onBackPressed() {
        Intent intent_back = new Intent(SettingsActivity.this, NewsHybridActivity.class);
        startActivity(intent_back);
    }
}

File: app_i/src/main/java/com/recoveryrecord/surveyandroid/example/activity/SettingsActivity.kt
package com.recoveryrecord.surveyandroid.example.activity

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import com.recoveryrecord.surveyandroid.example.NewsHybridActivity
import com.recoveryrecord.surveyandroid.example.R
import com.recoveryrecord.surveyandroid.example.ui.SettingsFragment

class SettingsActivity : AppCompatActivity() {
    @SuppressLint("CommitTransaction")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.settings_activity)
        if (savedInstanceState == null) {
            supportFragmentManager
                .beginTransaction()
                .replace(R.id.settings, SettingsFragment())
                .commit()
        }
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            onBackPressed()
            return true
        }
        return super.onOptionsItemSelected(item)
    }

    override fun onBackPressed() {
        startActivity(Intent(this@SettingsActivity, NewsHybridActivity::class.java))
    }
}
